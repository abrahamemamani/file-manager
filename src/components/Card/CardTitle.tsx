import { FC } from "react";

export interface CardTitleProps {
  value: string;
}

export const CardTitle: FC<CardTitleProps> = ({ value }) => (
  <span className="text-white text-center text-sm mt-5">{value}</span>
);
