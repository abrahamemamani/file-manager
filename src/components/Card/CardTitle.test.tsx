import { describe, expect, it } from "vitest";
import { CardTitle, CardTitleProps } from "./CardTitle";
import { render } from "@testing-library/react";

const defaultProps: CardTitleProps = {
  value: "New title",
};

describe("[components/Card]", () => {
  describe("when CardTitle is rendered", () => {
    it("should render", () => {
      const { asFragment } = render(<CardTitle {...defaultProps} />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
