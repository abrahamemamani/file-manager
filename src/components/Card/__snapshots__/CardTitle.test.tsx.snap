// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[components/Card] > when CardTitle is rendered > should render 1`] = `
<DocumentFragment>
  <span
    class="text-primary-gray dark:text-white text-center text-sm mt-5"
  >
    New title
  </span>
</DocumentFragment>
`;
